docs/lang/articles/contribution/contributor_guide.md

```diff
@@ -228,168 +228,62 @@ distinguish *what the user should know* from *what the
 developers are doing*. This is done by **capitalizing PR
 tags**:
 
-- PRs with visible/notable features to the users should be marked
-  with tags starting with **the first letter capitalized**, e.g.
+- PRs with visible or notable features to the users should be marked
+  with tags starting with **the first letter capitalized**, e.g.,
   `[Metal]`, `[Vulkan]`, `[IR]`, `[Lang]`, `[CLI]`. When releasing a new
-  version, a script (`python/taichi/make_changelog.py`) will
+  version, a script ([misc/make_changelog.py](https://github.com/taichi-dev/taichi/blob/master/misc/make_changelog.py)) will
   generate a changelog with these changes (PR title) highlighted.
   Therefore it is **important** to make sure the end-users can
   understand what your PR does, **based on your PR title**.
-- Other PRs (underlying development/intermediate implementation)
-  should use tags with **everything in lowercase letters**: e.g.
+- Other PRs (underlying development or intermediate implementation)
+  should use tags with **everything in lowercase letters**, e.g.,
   `[metal]`, `[vulkan]`, `[ir]`, `[lang]`, `[cli]`.
 - Because of the way the release changelog is generated, there
   should be **at most one capitalized tag** in a PR title to prevent
   duplicate PR highlights. For example,
-  `[GUI] [Mac] Support modifier keys` ([#1189](https://github.com/taichi-dev/taichi/pull/1189)) is a bad example, we
+  `[GUI] [Mac] Support modifier keys` ([#1189](https://github.com/taichi-dev/taichi/pull/1189)) is bad, and we
```

Comment:
maybe still keep "an example of bad tag choice"? "is bad" sounds like we are saying the PR is bad :P